### DISCUSSION TITLE
Is the next release the first beta version of EdgeDB and will it be ready for small personal projects?

### DISCUSSION THREAD

**Discussion by clmnin** [geldata/gel#1967] [Q&A] [answered]
As per the blog post for Alpha 6, @ambv mentions _Alpha 6 release is the sixth and final alpha release of EdgeDB 1.0_. And the team is getting ready for a public beta.

Will the public beta be ready for personal (but reliable) projects? Also, the next release data in up next week isn't it? Can't wait for the next release!

**Comment by 1st1** [comment, marked as answer]
Hi @clmno!

> Will the public beta be ready for personal (but reliable) projects?

That's what we're aiming for!

> Also, the next release data in up next week isn't it? Can't wait for the next release!

We will likely push the beta release 1-2 weeks later. There are a few things we must finalize before that.
